extensions/smallrye-openapi/spi/src/main/java/io/quarkus/smallrye/openapi/deployment/spi/OpenAPIBuildItem.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.smallrye.openapi.deployment.spi;
+
+import org.eclipse.microprofile.openapi.OASFilter;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class OpenAPIBuildItem extends MultiBuildItem {
+
+    private final OASFilter filter;
```

Comment:
Interesting that this works, I always thought we had to use simple types and build the object when the build item is consumed

Reply:
Not sure about that. But the test in Heath is working and creating the Document so at least it's working in test.